### PR TITLE
Add pop-Up window

### PR DESCRIPTION
**Pull request by Lusindiso**
### Here I have made these changes:
- Add a pop-up window html structure
- Add pop-up window JavaScript code
- Add dynamic data to JS file,
- Add JS code to display the dynamic data when the pop-up window is opened
- Add styles to style the pop-up window

### CODE REVIEW

**Review by nicupop729 — COMMENTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @Lusindiso, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that does not run linter check for JavaScript.
Please implement linters for JavaScript as well.
[here is the link for the correct linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: script.js [hunk 1]
```diff
@@ -1,25 +1,110 @@
...
+  },
+  project6: {
+    title: 'Website Protfolio',
+    text: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    lang: ['<li class="pop-up__languages--item">html</li>',
+      '<li class="pop-up__languages--item">Bootstrap</li>',
+      '<li class="pop-up__languages--item">Ruby</li>'],
+    image: './img/project-6.jpg',
+  },
+};
```
Comment (ghost) [inline script.js]: [REQUIRED] Nice job on storing all project-related data as `objects` within an `array` as it is required by this milestone :clap:. Now let's use this `data` to create all our project cards dynamically, right now I can see in your [index.html](https://github.com/Lusindiso/Microverse-Portfolio/blob/main/index.html) the project cards are still being created statically. Please refer to the below code lines:
https://github.com/Lusindiso/Microverse-Portfolio/blob/675ab3548909dbdaa57fe7b707769357bf768f0e/index.html#L102-L229

- [ ] Please kindly remove the above code from your [index.html](https://github.com/Lusindiso/Microverse-Portfolio/blob/main/index.html) and create each card dynamically with `JavaScript`.

To answer the question of why do we need to create cards dynamically, let's imagine you want to add a new project to your portfolio, with `static` code, you need to create another  `card ` in your HTML file and put all the data manually, meaning that you have to repeat yourself every time you want to add a new project, that is not good practice in terms of keeping your code `DRY`. `Dynamic code`, on the other hand, allows you to do it by just adding a new object to your `projects array` and the rest will be taken care of with the functions you create in your `script`. On top of that, this is one of the main requirements of this milestone, so let's create our project cards `dynamically`.

**_hints:_**

**Step-1**: Make sure the values of the project `objects` within the `array list` are correct.

**Step-2**: Select the main container for your cards, I think, it is section class `works`.

**Step-3**: Now we can iterate through our `array list`. You have many options like `for loop`, `forEach`, `map` and etc. You can go with either option depending on your convenience. I am gonna use `map` for this particular case. So we are looping the `array` each `element` is an object, in order to get the value from the object you can specify the appropriate key, something like `element.name`. Now keeping the above information in mind, we need to create an `html` element (project card) in each iteration and `append` it to our main container (mentioned in step-2).

_sample code:_
```JAVASCRIPT
const mainProjectsContainer = document.querySelector('.works');

mainProjectsContainer.innerHTML += myProjects.map((project) => {
return ( 
` <div class="projectCard">
        <img src=${project.image} alt="project screenshot">
        <div class="project-description">
          <h3 class="project-title">${project.title}</h3>
          <ul>
          // you can use the same logic to create `<li/>` tags dynamically with the array of techs
          </ul>
          <button type="button" class="see-project-btn">See this porject</button>
        </div>
      </div>
    </div>
  `)
}).join('');
```
**Step-4**: Make sure your static `project cards` from your [index.html](https://github.com/Lusindiso/Microverse-Portfolio/blob/main/index.html) are removed. I hope it is working as expected :pray:

_note:_  You can use the same logic with any other `loop`. Please note that the above code is not a direct solution, it is just a sample code to demonstrate the above-mentioned steps.

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @Lusindiso , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Lusindiso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**